src/Stratis.Bitcoin.Features.RPC.Tests/RPCSettingsTest.cs

```diff
@@ -57,14 +57,14 @@ public void Load_DefaultConfiguration_UsesDefaultNodeSettings()
             var nodeSettings = new NodeSettings(this.Network, args:new string[] { "-conf=" + confFile });
 
             var rpcSettings = new RpcSettings(nodeSettings);
-
-            Assert.False(rpcSettings.Server);
-            Assert.Equal(18332, rpcSettings.RPCPort);
-            Assert.Null(rpcSettings.RpcUser);
-            Assert.Null(rpcSettings.RpcPassword);
-            Assert.Empty(rpcSettings.Bind);
-            Assert.Empty(rpcSettings.DefaultBindings);
-            Assert.Empty(rpcSettings.AllowIp);
+            var defaults = new RpcSettings(NodeSettings.Default(this.Network));
+
+            Assert.Equal(defaults.Server, rpcSettings.Server);
```

Comment:
So now we are asserting against the defaults because it is pulled from the network? Wouldnt `rpcSettings` and `defaults` be exactly the same here anyway?

Reply:
We need to test that that is indeed the case when no config is provided.